src/test/java/de/tum/in/www1/artemis/exam/StudentExamIntegrationTest.java

```diff
@@ -173,12 +173,12 @@ class StudentExamIntegrationTest extends AbstractSpringIntegrationBambooBitbucke
 
     private final List<LocalRepository> studentRepos = new ArrayList<>();
 
-    private static final int NUMBER_OF_STUDENTS = 5;
+    private static final int NUMBER_OF_STUDENTS = 2;
 
     @BeforeEach
     void initTestCase() throws Exception {
-        // the service already creates 5 students, 1 tutor, 1 editor, 1 instructor
-        programmingExerciseTestService.setupTestUsers(TEST_PREFIX, 0, 0, 0, 1);
+        userUtilService.addUsers(TEST_PREFIX, 3, 1, 0, 2);
```

Comment:
The second argument is called `numberOfStudents` is there a reason why you set it to 3 and not used the constant `NUMBER_OF_STUDENTS`?

Reply:
There is a story behind this ^^ :

One of the tests previously needed a third student and therefor, I have set up the users with  `userUtilService.addUsers(TEST_PREFIX, NUMBER_OF_STUDENTS, 1, 0, 2);` and I have created the third user only for the one test that needed it. But after some testing, I realised that the additional creation of the third user was costing more time than I saved by reducing the amount of created students in the initTestCase() method. Therefor, I changed the number to 3, such that three students were created for each test and it became faster :O

Anyway, some commits later, I changed the test such that it does not require 3 students anymore. Looks like I forgot to undo my changes in the initTestCase() Method. I updated it just now :)